server/game/cards/01_SOR/DirectorKrennicAspiringToAuthority.ts

```diff
@@ -0,0 +1,29 @@
+import AbilityHelper from '../../AbilityHelper';
+import { LeaderUnitCard } from '../../core/card/LeaderUnitCard';
+
+export default class DirectorKrennicAspiringToAuthority extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8560666697',
+            internalName: 'director-krennic#aspiring-to-authority',
+        };
+    }
+
+    private buildKrennicAbilityProperties() {
+        return {
+            title: 'Give each friendly damaged unit +1/+0',
+            matchTarget: (card) => card.isUnit() && card.damage !== 0,
```

Comment:
Is this set up by default to only affect friendly units?

Reply:
Yes, and the tests confirm that